go/test/endtoend/onlineddl/vrepl/onlineddl_vrepl_test.go

```diff
@@ -818,36 +780,28 @@ func TestSchemaChange(t *testing.T) {
 	// - two shards as opposed to one
 	// - tablet throttling
 	t.Run("Revert a migration completed on one shard and cancelled on another", func(t *testing.T) {
-		// shard 0 will run normally, shard 1 will be throttled
-		defer unthrottleApp(shards[1].Vttablets[0], throttlerapp.OnlineDDLName)
-		t.Run("throttle shard 1", func(t *testing.T) {
-			body, err := throttleApp(shards[1].Vttablets[0], throttlerapp.OnlineDDLName)
-			assert.NoError(t, err)
-			assert.Contains(t, body, throttlerapp.OnlineDDLName)
-		})
+		// shard 0 will run normally, shard 1 will be postponed
 
 		var uuid string
-		t.Run("run migrations, expect 1st to complete, 2nd to be running", func(t *testing.T) {
-			uuid = testOnlineDDLStatement(t, alterTableTrivialStatement, "vitess", providedUUID, providedMigrationContext, "vtgate", "test_val", "", true)
+		t.Run("run migrations, expect running on both shards", func(t *testing.T) {
```

Comment:
I guess that most of the changes in this file helped to deflake the tests? Maybe it's another cherry-pick?

Reply:
With the new design, it's impossible to tell vitess "throttle one shard but not this other shard". Previously, it was possible since we hit a shard's `PRIMARY` tablet's HTTP API directly.

So the new design means we need to change the tests. And we don't need to test something that we don't support...